core-redirects-cae/src/main/java/com/tallence/core/redirects/cae/service/tasks/AbstractTask.java

```diff
@@ -62,9 +61,10 @@ boolean validate(Content redirect) {
     }
 
     Content targetLink = redirect.getLink(TARGET_LINK);
+    String targetUrl = redirect.getString(TARGET_URL);
 
-    if (targetLink == null) {
-      LOG.warn("redirect [{}] has no targetLink", redirect.getId());
+    if (targetLink == null && StringUtils.isEmpty(targetUrl)) {
```

Comment:
Noch ein Kandidat für eine `isEmpty()` am `Redirect`

Reply:
Aber hier hat man doch gar keinen Redirect in der Hand

Reply:
Oh, stimmt. Hab mich vom `redirect` irritieren lassen. Aber das ist ja nen Content